src/App.jsx

```diff
@@ -1,44 +1,82 @@
-import { useState } from "react"
-import {initialWorkouts, generateWorkout} from "./Workouts.js"
-import "./App.css"
+import { useState } from "react";
+import { initialWorkouts, generateWorkout } from "./Workouts.js";
+import "./App.css";
 
 function App() {
-  const [workouts, setWorkouts] = useState(initialWorkouts)
+  const [workouts, setWorkouts] = useState(initialWorkouts);
 
   const addNewWorkout = () => {
-    const newWorkout = generateWorkout()
-    console.log("addNewWorkout:", newWorkout)
-  }
+    const newWorkout = generateWorkout();
+    console.log("addNewWorkout:", newWorkout);
+    const addedWorkout = [...workouts, newWorkout];
```

Comment:
variable name is not good; should be updatedWorkoutsArray or something like that because `addedWorkout` tells me that the variable should represent 1 workout and it represents the one that was added...